apps/passport-server/src/services/telegramService.ts

```diff
@@ -118,7 +118,7 @@ export class TelegramService {
 
           await this.bot.api.sendMessage(
             userId,
-            `<i>Verifying and inviting...</i>`,
+            `<i>Verifying...\n\n\n\n\n\n\nInviting...</i>`,
```

Comment:
We don't have a great way to test this without pushing to staging, but I think ~7 new lines should be enough to avoid the toast

Reply:
What exactly is the toast?

Reply:
https://github.com/proofcarryingdata/zupass/assets/42984734/035cf549-43c2-4f0c-af37-d6beda5b9bd3

Reply:
Gotcha, from the video it looks like four lines should be more than enough. Can we switch to four?